PocketKit/Sources/PocketKit/Navigation/NavigationSidebarViewController.swift

```diff
@@ -113,6 +116,13 @@ class NavigationSidebarViewController: UIViewController {
         view = collectionView
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        let event = ImpressionEvent(component: .screen, requirement: .instant)
+        tracker.track(event: event, nil)
```

Comment:
This _could_ exist in its own view model, but that seems like a slightly overkill refactor for a 2-line fix.

Reply:
sounds reasonable to me too! Just an alt thought, what about adding tracker to `MainViewModel` and call `model.trackScreen()`